README.md

```diff
@@ -24,43 +24,120 @@ To keep the code base and the API simple, zerolog focuses on JSON logging only.
 * `net/http` helpers
 * Pretty logging for development
 
-## Usage
-
+## Installation
 ```go
-import "github.com/rs/zerolog/log"
+go get -u github.com/rs/zerolog/log
 ```
+## Getting Started
+### Simple Logging
+For simple logging, import the global logger package **github.co/rs/zerolog/log**
+```go
+package main
 
-### A global logger can be use for simple logging
+import (
+	"github.com/rs/zerolog/log"
+)
```

Comment:
You might make this one-line to reduce the size of the example and focus on the important stuff.

Reply:
No problem